src/scss/components/checkbox/mixins/_hxCheckboxControl.scss

```diff
@@ -8,22 +8,22 @@
     box-shadow: $focus-glow;
 
     &::-ms-check {
-      border-color: $cyan-700;
+      border-color: var(--hxCheckbox-bordercolor-focus, $cyan-700);
```

Comment:
please follow same pattern through out the application .

change it to var(--hxCheckbox-focus-bordercolor, $cyan-700);

Reply:
Changes have been done, Please review